test.py

```diff
@@ -0,0 +1,8 @@
+import cv2
```

Comment:
Is this still needed with image_grab_test.py?

Reply:
The idea is to use this file to get video streams from cameras and write the multi-processing code in here once we reach the R2 phase. I will rename this file to data_sender.py instead of test.py, but I believe that keeping this file would be better since it will used more later on.